fabric-chaincode-integration-test/src/test/java/org/hyperleder/fabric/shim/integration/DockerCompose.java

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+package org.hyperleder.fabric.shim.integration;
+import java.util.ArrayList;
+import java.util.List;
+
+/** Represents the 'peer' cli command
+ * 
+ * 
+ *
+ */
+public class DockerCompose extends Command {
+
+    public static DockerComposeBuilder newBuilder(){
+        return new DockerComposeBuilder();
+    }
+    
+    static public class DockerComposeBuilder extends Command.Builder<DockerCompose>{
+        String composeFile;
+
+        boolean up = true;
+        boolean detach = false;
+
+        public DockerComposeBuilder file(String composeFile){
+            this.composeFile = composeFile;
+            return this;
+        }
+        
+        public DockerComposeBuilder duplicate() {
+            return (DockerComposeBuilder) super.duplicate();
+        }
+
+        public DockerComposeBuilder up(){
+            this.up = true;
+            return this;
+        }
+        
+        public DockerComposeBuilder detach(){
+            this.detach = true;
+            return this;
+        }
+        
+        public DockerComposeBuilder down(){
+            this.up = false;
+            return this;
+        }
+      
+        public DockerCompose build(){
+
+            ArrayList<String> list = new ArrayList<>();
+            list.add("docker-compose");
```

Comment:
why are we launching this from Java?

Reply:
It could be launched from a script before - there's no specific reason. It fits in with the Junit testing.

The integration tests aren't good in either branch - and in master just couldn't get them to start at all.. so this is a transplant of the 1.4 ones.

Waiting in the wings for all the chaincodes is a replacement integration suite.